src/model/schemaModel.js

```diff
@@ -127,6 +127,10 @@ class SchemaModel {
 							});
 						}
 
+						if (schemaFlat[property].__type === 'date' && typeof operand === 'string') {
```

Comment:
I'll accept this to pass the merge request but we'll need to talk through the use-case for this.